lib/grammar.js

```diff
@@ -140,6 +140,12 @@ var grammar = module.exports = {
       reg: /^setup:(\w*)/,
       format: 'setup:%s'
     },
+    {
+      // a=connection:new
+      name: 'connectionType',
+      reg: /^connection:(new|existing)/,
```

Comment:
i'm slightly vary of hardcoding the enum variants here rather than using `/^connection:(.*)/`, but i suppose the RFC is clear